src/extra/libs/png/lv_png.c

```diff
@@ -272,4 +273,22 @@ static void convert_color_depth(uint8_t * img, uint32_t px_cnt)
 #endif
 }
 
+static inline lv_color_t lv_color_make_rounding(uint8_t r, uint8_t g, uint8_t b)
+{
+#if LV_COLOR_DEPTH == 16
+    /* if the high bits are not all set and the highest low bit is set     */
```

Comment:
In case of red 3 bits will be dropped. In value it's 0..7. Can't we just add 3 (half of 7) to round up the value after clipping in lv_color_make? (Handle overflow too, if course)